05_class_definition/01_class_definition_first_step.rb

```diff
@@ -39,8 +56,24 @@ class MetaClass
 #    ex.config = 'world'
 #    ExConfig.config #=> 'world'
 
-
 class ExConfig
+  class << self
+    def config=(value)
+      @config = value
+    end
+
+    def config
+      @config
+    end
```

Comment:
`attr_accessor`で代替可能。
```suggestion
    attr_accessor :config
```